docs/metadata.md

```diff
@@ -43,15 +43,15 @@ When you provide this information, cozy-client will automatically update metadat
 
 ### For new documents
 
-Cozy-Client will set: 
+Cozy-Client will set:
 
 * `metadataVersion` <string>: to the last version known to Cozy-Client. It's the version of the schema for the metadata block itself.
 * `doctypeVersion` <string>: if it's provided in the `schema` at Cozy-Client initialization
 * `slug` <string>: Slug of the app
-* `sourceAccount`: 
+* `sourceAccount`:
```

Comment:
You probably can remove it, since there is no description and you mention it afterwards

Reply:
I'm not sure about it, because you can still pass a sourceAccount via the constructor of a client in the cozy-client source code. (ex: for other doctype)

Reply:
Thus maybe a description like this: 
``` 
`sourceAccount`: <string>: if it's provided in the `appMetadata` at initialization. Note it won't work for files (see below).
```